Configuration/PyReleaseValidation/python/relval_production.py

```diff
@@ -12,6 +12,8 @@
 workflows[1000] = [ '',['RunMinBias2011A','TIER0','SKIMD','HARVESTDfst2','ALCASPLIT']]
 workflows[1001] = [ '',['RunMinBias2011A','TIER0EXP','ALCAEXP','ALCAHARVDSIPIXELCALRUN1','ALCAHARVD1','ALCAHARVD2','ALCAHARVD3','ALCAHARVD4','ALCAHARVD5','ALCAHARVD7','ALCAHARVD8']]
 workflows[1001.2] = [ '',['RunZeroBias2017F','TIER0EXPRUN2','ALCAEXPRUN2','ALCAHARVDSIPIXELCAL','ALCAHARVDSIPIXELCALLA','ALCAHARVD4','ALCAHARVDSIPIXELALIHG']]
+workflows[1001.3] = [ '',['RunSingleMuon2022A','TIER0EXPRUN3','ALCAEXPRUN3','ALCAHARVDEXPRUN3']]
+workflows[1002.3] = [ '',['RunZeroBias2022B','TIER0PROMPTRUN3','ALCAEXPRUN3','ALCAHARVDEXPRUN3']]
```

Comment:
why is the "Prompt" configuration re-running the `ALCAEXPRUN3` block? Shouldn't it rather run the run-3 equivalent of `ALCASPLIT` ?